src/offline/CachePostLoginAction.ts

```diff
@@ -15,27 +13,24 @@ export class CachePostLoginAction implements IPostLoginAction {
 		private readonly entityClient: EntityClient,
 		private readonly progressTracker: ProgressTracker,
 		private readonly logins: LoginController,
-		private readonly systemApp: NativeSystemApp | null
 	) {
 
 	}
 
 	async onFullLoginSuccess(loggedInEvent: LoggedInEvent): Promise<void> {
-		if (await this.systemApp?.getConfigValue(DesktopConfigKey.offlineStorageEnabled)) {
-			// 3 work to load calendar info, 2 work to load short and long events
-			const workPerCalendar = 3 + 2
-			const totalWork = this.logins.getUserController().getCalendarMemberships().length * workPerCalendar
-			const monitorHandle = this.progressTracker.registerMonitor(totalWork)
-			const progressMonitor = this.progressTracker.getMonitor(monitorHandle)
-			const calendarInfos = await this.calendarModel.loadCalendarInfos(progressMonitor!)
-			const loadingPromises = []
-			for (const {groupRoot} of calendarInfos.values()) {
-				loadingPromises.push(this.entityClient.loadAll(CalendarEventTypeRef, groupRoot.longEvents, CUSTOM_MIN_ID).then(() => progressMonitor?.workDone(1)))
-				loadingPromises.push(this.entityClient.loadAll(CalendarEventTypeRef, groupRoot.shortEvents, CUSTOM_MIN_ID).then(() => progressMonitor?.workDone(1)))
-			}
-			await Promise.all(loadingPromises)
-			progressMonitor?.completed()
+		// 3 work to load calendar info, 2 work to load short and long events
+		const workPerCalendar = 3 + 2
+		const totalWork = this.logins.getUserController().getCalendarMemberships().length * workPerCalendar
+		const monitorHandle = this.progressTracker.registerMonitor(totalWork)
+		const progressMonitor = this.progressTracker.getMonitor(monitorHandle)
```

Comment:
should unwrap here instead of soft-calling it later

Reply:
there's a second commit...